test/libethereum/Block.cpp

```diff
@@ -50,6 +50,8 @@ BOOST_AUTO_TEST_CASE(bStructures)
 
 BOOST_AUTO_TEST_CASE(bStates)
 {
+	if (!dev::test::Options::get().quadratic)
```

Comment:
cmake scans test files for boost test suites and then it all gets executed.
`./testeth -t <suitename>/<casename> `

some tests are disabled by default. so you could run it by
`./testeth --memory --quadratic --wallet` 

or just use all option
`./testeth --all`